app/models/concerns/djatoka_adapter.rb

```diff
@@ -2,14 +2,8 @@
 # Djatoka-backed implementation of StacksImage delivery
 module DjatokaAdapter
   def response
-    return to_enum(:response) unless block_given?
-
     benchmark "Fetch #{url}" do
-      client.get(url) do |req|
-        req.on_body do |_, chunk|
-          yield chunk
-        end
-      end
+      HTTP.get(url).body
```

Comment:
`HTTP::Reponse#body` streams the response back as an IO-like object. Faraday doesn't (yet) support content streaming.